etc/behat/suites/ui_promotion.yml

```diff
@@ -19,6 +19,9 @@ default:
                 - sylius.behat.context.setup.promotion
                 - sylius.behat.context.setup.shipping
                 - sylius.behat.context.setup.zone
+                - sylius.behat.context.transform.lexical
```

Comment:
I think transforms should not be mixed with setup contexts. I will increase readability.